recipes/localhgt/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: localhgt
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("localhgt", max_pin="x") }}
+  entry_points:
+    - localhgt = localhgt:main
+
+source:
+  url: https://github.com/deepomicslab/LocalHGT/archive/refs/tags/v{{ version }}.tar.gz
+  md5: 8438fdd9cb3ad02fe331a09d4ca0ccfe
```

Comment:
Old recipes use `md5` but `sha256` is recommended for newer recipes.

Reply:
I revised this as suggested.